### PR TITLE
#365 Require ISSN early in the PLN setup process

### PR DESCRIPTION
**Pull request by ubermichael**
Require journal managers to provide an ISSN before allowing them to agree to the terms of use. 

Fixes pkp/pkp-lib#365

### CODE REVIEW

File: plugins/generic/pln/classes/form/PLNSettingsForm.inc.php [hunk 1]
```diff
@@ -74,7 +74,19 @@ function readInputData() {
...
 	 */
 	function display() {
+		$journal =& Request::getJournal();
+		$issn = '';
+		if ($journal->getSetting('onlineIssn')) {
+			$issn = $journal->getSetting('onlineIssn');
+		} else if ($journal->getSetting('printIssn')) {
+			$issn = $journal->getSetting('printIssn');
+		}
+		$has_issn = false;
```
Comment (asmecher) [inline plugins/generic/pln/classes/form/PLNSettingsForm.inc.php]: Just a stylistic thing -- and I see it's probably elsewhere in this plugin -- but we generally use lowerCamelCase in variable names.

File: plugins/generic/pln/templates/settings.tpl [hunk 2]
```diff
@@ -24,11 +24,15 @@
...
 				<td class="value">
 					<p>{translate key="plugins.generic.pln.settings.terms_of_use_help"}</p>
-					{foreach name=terms from=$terms_of_use key=term_name item=term_data}
-					<p>{$term_data.term}</p>
-					<input type="checkbox" name="terms_agreed[{$term_name|escape}]" value="1"{if $terms_of_use_agreement[$term_name] == 1} checked{/if}><label class="agree" for="terms_agreed[{$term_name|escape}]">{translate key="plugins.generic.pln.settings.terms_of_use_agree"}</label>
-					{if !$smarty.foreach.terms.last }<div class="separator">&nbsp;</div>{/if}
-					{/foreach}
+					{if $has_issn}
+						{foreach name=terms from=$terms_of_use key=term_name item=term_data}
+						<p>{$term_data.term}</p>
```
Comment (asmecher) [inline plugins/generic/pln/templates/settings.tpl]: Another stylistic thing -- another indent for the foreach would make it more readable IMO.

**Comment by asmecher** [comment]
Two comments, Michael, but both minor. I'm ready to merge -- just let me know.

**Comment by ubermichael** [comment]
Stylistic changes made and pushed. Merge if you see fit.

**Comment by asmecher** [comment]
Looks good, thanks!

**Pull request merged**